packages/fether-react/src/Accounts/CreateAccount/AccountImportOptions/AccountImportOptions.js

```diff
@@ -28,26 +33,67 @@ class AccountImportOptions extends Component {
       location: { pathname }
     } = this.props;
     const currentStep = pathname.slice(-1);
+
     history.push(`/accounts/new/${+currentStep + 1}`);
   };
 
   handlePhraseChange = ({ target: { value: phrase } }) => {
+    const words = phrase.split(' ');
+    const lastVal = words.slice(-1);
+    const isWordEnded = lastVal.join() === '';
+
+    let lastWord;
+    if (isWordEnded) {
+      lastWord = words[words.length - 2];
+    }
+
+    if (
+      isWordEnded &&
+      lastWord &&
+      !BIP39_WORDLIST.has(lastWord) &&
+      !PARITY_WORDLIST.has(lastWord)
+    ) {
+      // Guide them to contact Fether Riot
+      let error = (
+        <div>
+          {`${lastWord} is not a valid BIP39 or Parity word. If you wish to recover your account with a self-generated phrase, please`}
+          <button className='button -utility'>
+            <a
+              className='contact'
+              href='https://riot.im/app/#/room/#fether:matrix.parity.io'
+              rel='noopener noreferrer'
+              target='_blank'
+            >
+              Contact Us
+            </a>
+          </button>
+        </div>
+      );
+
+      this.setState({
+        error
+      });
+    } else {
+      this.setState({
+        error: null
+      });
+    }
+
     this.setState({ phrase });
   };
 
   handleSubmitPhrase = async () => {
     const phrase = this.state.phrase.trim();
     const {
-      createAccountStore,
-      createAccountStore: { setPhrase }
+      createAccountStore: { address, setPhrase }
     } = this.props;
 
     this.setState({ isLoading: true, phrase });
 
     try {
       await setPhrase(phrase);
 
-      if (this.hasExistingAddressForImport(createAccountStore.address)) {
+      if (this.hasExistingAddressForImport(address)) {
```

Comment:
revert this change; makes it impossible to retrieve from a seed phrase (throws an error). see other comment